clients/cloud/java/pom.xml

```diff
@@ -34,22 +40,18 @@
     <repositories>
         <repository>
             <id>confluent</id>
+						<name>Confluent</name>
```

Comment:
Is the spacing off here?

Reply:
yeah, tabs snunk in there.  Fixed.  We should look into https://editorconfig.org/